### PR TITLE
Changes for integration of qlora into CLLM training

### PR DESCRIPTION
**Pull request by astradzhao**
This branch adds functionality for qLoRA into the CLLM training script, with set hyperparameter of rank = 32, lora_alpha = 16, dropout = 0.05.

### CODE REVIEW

**Comment by snyhlxde1** [comment]
Please make qlora training configurable:
add an optional training argument (a boolean flag, --qlora, taking on argument True/False) to the [training script](https://github.com/hao-ai-lab/Consistency_LLM/blob/main/cllm/train_cllm_global.py).
depending on whether this flag is set, turn on qlora training (feel free to set it as default, i.e., default argument is True).

**Comment by astradzhao** [comment]
> Please make qlora training configurable: add an optional training argument (a boolean flag, --qlora, taking on argument True/False) to the [training script](https://github.com/hao-ai-lab/Consistency_LLM/blob/main/cllm/train_cllm_global.py). depending on whether this flag is set, turn on qlora training (feel free to set it as default, i.e., default argument is True).

Okay, fixed!
